oemoflex/model_structure/foreign_keys.yml

```diff
@@ -0,0 +1,55 @@
+'bus': [
+    'heat-shortage',
+    'heat-excess',
```

Comment:
The format is a bit upside down in my view, even though congruent with oemof.tabular. Adding a component will probably be error-prone. We should keep in mind that this should be resolved some day.

I would rather prefer something like this:
```yaml
uranium-nuclear-st:
  foreign-keys:
  - from_to_bus

wind-offshore:
  foreign-keys:
  - bus
  - profile
```

Reply:
To convert it I tried this:
```python
y = yaml.safe_load(open('oemoflex/model_structure/foreign_keys.yml', 'r'))
d = {}
for key, list in y.items():
    for i in list:
        d.setdefault(i, {'foreign-keys':[]})['foreign-keys'].append(key)
```
Since the resulting yaml deviates from the tabular format, the corresponding `inferring()` function to read in the yaml file and convert it to the tabular format is not easier to read. I gave it a try...